core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -462,7 +536,7 @@ private void applyOutstandingWalTransactions(
                 // be returned to the pool and dirty writes will be rolled back. We have to update the sequencer
                 // on the state of the writer and revert any dirty txns that might have advanced. We do that
                 // by equalizing writerTxn and dirtyWriterTxn.
-                engine.getTableSequencerAPI().updateWriterTxns(tableToken, writer.getSeqTxn(), writer.getSeqTxn());
+                engine.getTableSequencerAPI().updateWriterTxns(tableToken, writer.getSeqTxn(), writer.getAppliedSeqTxn());
```

Comment:
nit: could you check the comment for this line and may be elaborate what "equalizing" means?